### ISSUE TITLE
Enhancement: un-force landscape mode

### ISSUE THREAD

**Issue by squshy7** [jlucidar/fpv_viewer_android#20]
As per title, default app behavior should follow global orientation settings, and as such allow auto rotate. This is mostly UX related as while I expect the vast majority of users to be using it in landscape, it is a jarring experience to have an app that's not a game force you to turn your phone before you are ready.

Not urgent, recommend for post-beta milestone.

**Comment by jlucidar** [comment]
Well, isn't fpv sort of a game ? :p
Let's see what people think about this :)

**Comment by fichek** [comment]
I don't think it makes sense to enable portrait mode for an app that shows a fullscreen video that's always landscape and will show nothing more than big black bars below/above if it's in portrait, but it definitely needs `sensorLandscape` at least to be able to rotate to the other landscape. I tried it and nothing seems to break.

**Comment by squshy7** [comment]
> I don't think it makes sense to enable portrait mode for an app that shows a fullscreen video that's always landscape and will show nothing more than big black bars below/above if it's in portrait, but it definitely needs `sensorLandscape` at least to be able to rotate to the other landscape. I tried it and nothing seems to break.

Consider:
1.) If there ends up being any appreciable amount of settings in a settings menu, it is a much better UX to _be able_ to handle these in portrait

2.) Tablets have more than enough screen real estate for portrait to be an acceptable viewing orientation; take in to consideration those tablets whose aspect ratio makes them uncomfortable in landscape

**Comment by fichek** [comment]
I'd definitely let settings screen rotate to any orientation, not a problem since it's a separate activity, just not convinced about main video player activity needing portrait. I feel like it will just complicate things like #18 - should the overlay be placed over the actual video frame or just bottom left of the entire screen which will just show it over the black bars?

**Comment by D3VL-Jack** [comment]
Personally I'd vote the water mark goes in the bottom left of the video frame, this way if in 4:3 and landscape you won't end up with the potential of having half the watermark over black and the other half over video.

**Comment by fichek** [comment]
> Personally I'd vote the water mark goes in the bottom left of the video frame, this way if in 4:3 and landscape you won't end up with the potential of having half the watermark over black and the other half over video.

It's worse than that, 100% of the watermark is over black on most phones today.

![v1](https://cdn.discordapp.com/attachments/841708139740397610/843146024889614356/Screenshot_20210515-171944.png)

![v2](https://cdn.discordapp.com/attachments/841708139740397610/843146024502427749/Screenshot_20210515-172017.png)
